README.md

```diff
@@ -1,2 +1,429 @@
-# iceberg-catalog-migrator
-CLI tool to bulk migrate the tables from one catalog another without a data copy
+# Objective 
+Introduce a command-line tool that enables bulk migration of Iceberg tables from one catalog to another without the need to copy the data.
+
+There are various reasons why users may want to move their Iceberg tables to a different catalog. For instance,
+* They were using hadoop catalog and later realized that it is not production recommended. So, they want to move tables to other production ready catalogs.
+* They just heard about the awesome Arctic catalog (or Nessie) and want to move their existing iceberg tables to Dremio Arctic.
+* They had an on-premise Hive catalog, but want to move tables to a cloud-based catalog as part of their cloud migration strategy.
+
```

Comment:
A brief discription of the two different "modes" should be added in the first chapter